### DISCUSSION TITLE
Machine-readable form of attributes

### DISCUSSION THREAD

**Discussion by lhmarsden** [cf-convention/discuss#398] [Comments and ideas for changing CF]
### Topic for discussion

I would like there to be a machine-readable form of the attributes in Appendix A of the CF conventions:
https://github.com/cf-convention/cf-conventions/blob/main/appa.adoc

I am making developments to the Nansen Legacy template generator, and would like to be able to add a page where the user can add variable attributes. For effective maintainence over time, it would be good if I was reading these attributes from some machine-readable online source. I suppose this could be useful for other tools/software that support CF too.

Does this exist? If not, I am willing to do the leg work to develop this. But it would be good to hear people's thoughts on how this should be done and what format people would like this to be in. 

I was imagining something similar to how the CF standard names are structured:
https://cfconventions.org/Data/cf-standard-names/current/src/cf-standard-name-table.xml

**Comment by feggleton** [comment]
I added the CF Attributes in Appendix A into the NERC Vocab Server last year https://vocab.nerc.ac.uk/collection/P31/current/

However, note that this has not been updated since so if you find any that aren't on there, please let me know and I can get them added in. I haven't been monitoring when new attributes are added. Hope this helps.

**Reply by lhmarsden** [reply]
Thanks, Fran. Your reply reminded me that I asked this already some months ago. Sorry for repeating! I will close this now.

https://github.com/orgs/cf-convention/discussions/384
